savoirfairelinux/sous-chef
# Add more member update steps.

## Fixes #393

### Changes proposed in this pull request:

* Add payment_information update step.
* Add referencing update steps.
* Add emergency_contact update step.

### Status


### How to verify this change

*When viewing the client information, click on "Edit" and update any information you want, then go back to see the same client and verify that the information is updated.*

### Additional notes

Remove member's duplicate template.
Add base class to client's update and view views.
Use a base class to group same code from "update client" views,
"view client" views and "update client" tests.
Remove duplicated templates to update member information, reuse the
template that exist already to add new members.

Ref: issue #393
